plugins/lua/ts_lua_util.c

```diff
@@ -311,6 +311,69 @@ ts_lua_del_module(ts_lua_instance_conf *conf, ts_lua_main_ctx *arr, int n)
   return 0;
 }
 
+int
```

Comment:
what do you mean by "recreating the context"?

Reply:
The `arr` array - it must have been created at some point, so an alternative is rather than running this code on the `arr[i]` instance, do whatever was done to set the value in `arr[i]` originally.

Reply:
That's pretty much the same thing.